server/migrations/20190111125359-create-role.js

```diff
@@ -1,26 +1,31 @@
 export default {
-  up: (queryInterface, Sequelize) => queryInterface.createTable('Roles', {
-    id: {
-      allowNull: false,
-      autoIncrement: true,
-      primaryKey: true,
-      type: Sequelize.INTEGER
-    },
-    type: {
-      type: Sequelize.STRING,
-      allowNull: false,
-      unique: true
-    },
-    createdAt: {
-      allowNull: false,
-      type: Sequelize.DATE,
-      defaultValue: Sequelize.literal('CURRENT_TIMESTAMP')
-    },
-    updatedAt: {
-      allowNull: false,
-      type: Sequelize.DATE,
-      defaultValue: Sequelize.literal('CURRENT_TIMESTAMP')
-    }
-  }),
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.query('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";')
```

Comment:
What's happening on this Line?

Reply:
Don't forget to close all open conversations when resolved

Reply:
@jesseinit the extension is used to autogenerate uuids upon row insertion